i18n/fr.json

```diff
@@ -0,0 +1,59 @@
+{
+  "menu": {
+      "exit": "QUITTER"
+    , "help": "AIDE"
+    , "completed": "FAIT"
+    , "language": "CHOISIR LA LANGUE"
+    , "cancel": "ANNULER"
+  }
+, "language": {
+      "_current": "ACTUEL"
+    , "en": "Anglais (English)"
+    , "fr": "Français"
+    , "ja": "Japonais (日本語)"
+    , "zh-cn": "Chinois (中文)"
+    , "zh-tw": "Taiwanais (國語)"
+    , "es": "Espagnol (Español)"
+    , "pt-br": "Portugais brésilien (Português)"
+  }
+, "error": {
+      "exercise": {
+          "none_active": "Aucun exercice en cours.  Choisissez-en un dans le menu."
+        , "missing": "L’exercice {{{name}}} est introuvable."
+        , "unexpected_error": "Impossible de {{mode}} : {{{err}}}"
+        , "missing_file": "ERREUR : {{{exerciseFile}}} n’existe pas !"
+        , "not_a_workshopper": "ERREUR : {{{exerciseFile}}} n’est pas un exercice d’atelier"
+        , "preparing": "Erreur à la préparation de l’exercice : {{{err}}}"
```

Comment:
If it's not too long, "Erreur lors de la préparation de l'exercice : {{{err}}}" sounds more accurate.
If a short sentence is needed, "Erreur en préparant l'exercice : {{{err}}}" sounds better.